tests/Feature/Modules/Invoices/OrderToUpsellCrudTest.php

```
@@ -0,0 +1,158 @@
+            ->test(ListOrderToUpsells::class)
+            ->assertCanSeeTableRecords(
+                Order::where('company_id', $this->company->id)
+                    ->where('invoice_status', InvoiceStatus::UPSELLING)
+                    ->get()
+            )
+            ->assertCanNotSeeTableRecords(
+                Order::where('company_id', $this->company->id)
+                    ->where('invoice_status', '!=', InvoiceStatus::UPSELLING)
+                    ->get()
+            );
+    }
```

PHPMD (2.15.0)

The method it_lists_orders_to_upsell is not named in camelCase. (undefined)

(CamelCaseMethodName)